### PR TITLE
Fix account changed to primary after being refresh

### PR DESCRIPTION
**Pull request by Ghost-Slime**

### CODE REVIEW

File: src/modules/account/components/AccountTile.vue [hunk 1]
```diff
@@ -195,11 +195,34 @@ export default{
         return
       }
     }
+
+    const setDefaultAccInStorage = (address) =>{
+      let data = [{
+            walletName: walletState.currentLoggedInWallet.name,
+            defaultAcc: address
+          }]
```
Comment (sleepyOwl14) [inline src/modules/account/components/AccountTile.vue:203]: why it is in array ? it does not need to use array, just object

Comment (Ghost-Slime) [reply src/modules/account/components/AccountTile.vue:203]: Noted

Comment (Ghost-Slime) [reply src/modules/account/components/AccountTile.vue:203]: Noted

**Comment by sleepyOwl14** [comment]
no visible issue on code, just maybe can reuse the same method/function `setDefaultAccInStorage` as it is duplicated in everywhere

**Comment by Ghost-Slime** [comment]
> no visible issue on code, just maybe can reuse the same method/function `setDefaultAccInStorage` as it is duplicated in everywhere

Noted

**Review by anakin001 — APPROVED** [review]

**Pull request merged**